### PR TITLE
Fix idle timeout [PROD-537]

### PR DESCRIPTION
**Pull request by zarsky-broad**
Verified in-browser.

We now ensure that signing back in to your google account in another tab won't sign you back in to Terra.

### CODE REVIEW

**Review by petesantos — APPROVED** [review]
Paired and reviewed

File: src/components/IdleStatusMonitor.js [hunk 1]
```diff
@@ -35,30 +37,45 @@ const IdleStatusMonitor = ({
   timeout = Utils.durationToMillis({ minutes: 15 }),
   countdownStart = Utils.durationToMillis({ minutes: 3 })
 }) => {
+  // State
```
Comment (dmohs) [inline src/components/IdleStatusMonitor.js:40]: I judge comments like this do not add enough value to justify them.

**Review by dmohs — APPROVED** [review]

File: src/components/IdleStatusMonitor.js (hunk 1 above)
Comment (zarsky-broad) [reply src/components/IdleStatusMonitor.js:40]: I've been adding these indicators in this order with this spacing to all hook-based components, and I've consistently received feedback that they're useful to people in lieu of the class-based methods that previously served as waypoints in the component's structure.

**Comment by zarsky-broad** [comment]
Test failures are known flake, merging.

**Pull request merged**